src/box/iproto.cc

```diff
@@ -1528,6 +1713,9 @@ static void
 tx_process1(struct cmsg *m)
 {
 	struct iproto_msg *msg = tx_accept_msg(m);
+
+	struct txn *txn = tx_prepare_transaction_for_request(msg->txn);
```

Comment:
in release it will be unused variable because assert will be removed (see build log on pipeline)